app/emails/briefs.py

```diff
@@ -624,6 +624,54 @@ def send_brief_clarification_to_seller(brief, brief_question, to_address):
         db_object=brief)
 
 
+def send_opportunity_closed_early_email(brief, current_user):
+    # to circumvent circular dependencies
+    from app.api.services import audit_service, audit_types
+
+    audit_event = audit_service.find(type=audit_types.sent_closed_brief_email.value,
+                                     object_type="Brief",
+                                     object_id=brief.id).count()
+
+    if (audit_event > 0):
+        return
```

Comment:
Should we remove this audit check?  If an Ops request comes through to open the opportunity after an accidental close, would this prevent further emails from being sent if it's closed early again?

Reply:
i don't think this is needed, as it should be a 1 time event